dracut/modules.d/99asahi-firmware/module-setup.sh

```diff
@@ -20,7 +20,7 @@ depends() {
 
 # called by dracut
 installkernel() {
-    instmods apple-mailbox nvme-apple
+    instmods apple-mailbox nvme-apple vfat
```

Comment:
This is the wrong file to put it in. It should go into `10-asahi.conf` instead.

Reply:
What in the distinction between putting modules in here and 10-asahi.conf ? Just curious seeing as apple-mailbox and nvme-apple is duplicated between both. Normally in other cases I end up putting every kernel module I need in "installkernel" in the module-setup.sh file but I'm curious what the distinction is here.

Reply:
The stuff that's in this file are for guaranteeing the logic this module needs to run. If you want to guarantee arbitrary modules are included, they should be in the config file instead.

Reply:
Can this dracut module run without vfat?

Reply:
Hmm, I guess not, since it needs to mount the ESP.

Reply:
I think this is the correct place. `load-asahi-firmware.sh` actively mounts the ESP partition. If the asahi-firmware module is not used there is no need to include vfat.

Reply:
You're right.